### DISCUSSION TITLE
Getting all peaks from a cell type pseudobulk

### DISCUSSION THREAD

**Discussion by joseverdezoto** [GreenleafLab/ArchR#1368] [Questions / Documentation]
Hello,

Thanks so much for this great package. I had a question regarding peak calling. Is there a way to extract all the peaks that were called by each individual cell type pseudobulk? Using `getPeakSet` returns the union peak set but each peak is annotated with the cell that has the highest significance for that peak. We're interested in all of the peaks that were called per single cell type, regardless of whether they are present in other ones. 

Thanks!

**Comment by rcorces** [comment]
https://github.com/GreenleafLab/ArchR/discussions/1301
